CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## devel
 
+### Bug Fixes:
+#### Modules:
+  - zabbix_action - now correctly selects mediatype for the (normal|recovery|update) operations with Zabbix 4.4 and newer.
```

Comment:
How about add at the end the related for PR or issue link?

Reply:
Yes, I was counting on rebasing this either way due to #92. Btw we are already hitting conflicts in CHANGELOG.md so maybe we should back off on requirement to add CHANGELOG entries and just sum them up before release.